packages/controlled-form/src/index.ts

```diff
@@ -10,3 +10,50 @@ export * from './lib/ProviderAutocomplete';
 export * from './lib/RadioGroup';
 export * from './lib/Select';
 export * from './lib/TextField';
+
+export {
+  FormProvider,
+  /** Custom react-hook-form hook to manage the entire form.
```

Comment:
Would this still work if it's put with the component declaration? It feels a little out of place here

Reply:
it felt weird putting it in other component files when we're not changing/wrapping anything. I originally had it in the ControlledForm file, but if we're deprecating/removing ControlledForm then I don't see putting in it's own or another file so exporting it with everything else felt like the next best thing.